Watchman.Web/Jobs/Statsman/CalculateMessagesPerDay.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Models;
+using Statsman.Core.Generators;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Watchman.Web.Jobs.Statsman
+{
+    public class CalculateMessagesPerDay : IhangfireJob
+    {
+        private readonly DiscordServersService discordServersService;
+        private readonly PreReneratedStatisticsGenerator preReneratedStatisticsGenerator;
+
+        public CalculateMessagesPerDay(DiscordServersService discordServersService, PreReneratedStatisticsGenerator preReneratedStatisticsGenerator)
+        {
+            this.discordServersService = discordServersService;
+            this.preReneratedStatisticsGenerator = preReneratedStatisticsGenerator;
+        }
+
+        public RefreshFrequent Frequency => RefreshFrequent.Daily;
+
+        public async Task Do()
+        {
+            await foreach (var server in this.discordServersService.GetDiscordServersAsync())
```

Comment:
pytanie czy zwykły linq nie będzie tutaj lepszy?
`var tasks = await this.discordServersService.GetDiscordServersAsync().SelectAsync(x => this.preReneratedStatisticsGenerator.PreGenerateStatisticsPerDay(x.Id).ToArrayAsync();`

Reply:
(to samo do pozostałych klas)

Reply:
myśle że foreach będzie czytelniejszy w tym przypadku